abicheck/dumper.py

```
@@ -635,6 +635,25 @@ def _clang_fallback_ready() -> bool:
     return "castxml-direct-include-guard"
 
 
+def _configured_target_triple(
+    gcc_options: str | None, gcc_option_tokens: tuple[str, ...], clang_bin: str
+) -> str | None:
+    """Return the target reported by configured Clang and its pass-through flags."""
+    cmd = [clang_bin, *split_gcc_options(gcc_options or ""), *gcc_option_tokens, "-print-target-triple"]
+    try:
+        result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)